00-Raster-Structure.Rmd

```diff
@@ -373,7 +387,7 @@ plotRGB(RGB_stack, r = 1, g = 2, b = 3,
 
 ```
 
-Below - the black edges have been assigned NoDataValues (`NA`). R doesn't render 
+Below - the black edges have been assigned NoDataValues (`NA`). R doesn't render
```

Comment:
NoDataValue  is used.  I've go through the section and changed to `NoDataValue` where appropriate.